adam-core/src/main/scala/org/bdgenomics/adam/util/FileExtensions.scala

```diff
@@ -118,4 +118,28 @@ object FileExtensions {
       pathName.endsWith(".vcf.bgzf") ||
       pathName.endsWith(".vcf.bgz")
   }
+
+  /**
+   * @param pathName The path name to match.
+   * @return Returns true if the path name matches an HTSJDK sequence dictionary (.dict) extension.
+   */
+  private[adam] def isDictExt(pathName: String): Boolean = {
```

Comment:
Small nit, but instead of making all the methods `private[adam]`, can we make the object `private[adam]`.